conf/formTemplateSchema.json

```diff
@@ -965,6 +965,19 @@
               "pattern": "^postcodeLookup$"
             }
           }
+        },
+        "prefix": {
+          "required": [
+            "type"
+          ],
+          "properties": {
+            "type": {
+              "pattern": "^text$"
+            },
```

Comment:
I've added the requirement for the type field since this is the case, although it is not on the ticket. Happy to remove if necessary.